.github/workflows/pr-build.yml

```diff
@@ -28,6 +28,6 @@ jobs:
       - name: Install and Build 🔧
         run: |
           bundle install
-          npm ci
+          npm install
```

Comment:
`npm ci` requires `package-lock.json` to be updated to the latest version of `@loopback/docs`.